run/README.namelist

```diff
@@ -1562,6 +1562,7 @@ The following are for observation nudging:
                                      = .true. , ! use microphysics variables in bdy file
  have_bcs_scalar (max_dom)           = .false., ! model run after ndown only: do not use scalar variables in bdy file
                                      = .true. , ! use scalar variables in bdy file
+ multi_bdy_files                     = .false., ! F=default; T=model will run with split and individually named LBC files
```

Comment:
@davegill Notes put in examples.namelist should be included here. I'm particularly interested in the part that states that when this option is used, only one time period bdy file can be used.
Also is 'multi_bc_files' a better namelist variable to use?

Reply:
@weiwangncar 
Wei,
I have added info to the run/README.namelist file